### DISCUSSION TITLE
Replace .df attribute in TimeSeriesMixin with .to_pandas() and .to_polars() methods

### DISCUSSION THREAD

**Discussion by aleewen** [alpacahq/alpaca-py#389] [Ideas]
Hi all,

Having .df as an attribute will slow things down for those who do not want it. Instead, can we replace it with .to_pandas() and .to_polars() methods? This will also allow us to add in Polars support in the process.

**Comment by impredicative** [comment]
Well, `.df` can in principle be made dynamic if it isn't already, so it doesn't then slow anyone down. As for polars, tropics, meridians, etc. it's best to do your own custom processing.
